lib/workos/user_management.rb

```diff
@@ -462,6 +462,40 @@ def authenticate_user_with_code(code:, client_id:, ip_address: nil, user_agent:
         )
         WorkOS::UserResponse.new(response.body)
       end
+      #
```

Comment:
Can we leave an extra blank line above this comment block like we've done for other methods?

```suggestion

```